projects/lib-magnetar-quill/src/lib/components/table-modal/table-modal.component.html

```
@@ -0,0 +1,18 @@
+<div class="modal-overlay">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)